_episodes/02-numpy.md

```diff
@@ -393,6 +393,11 @@ standard deviation: 4.61383319712
 > to see a list of all functions and attributes that you can use. After selecting one, you
 > can also add a question mark (e.g. `numpy.cumprod?`), and IPython will return an
 > explanation of the method! This is the same as doing `help(numpy.cumprod)`.
+> Similarly, if you are using the "plain vanilla" Python interpreter, you can type `numpy.` 
+> and press the Tab key twice for a listing of what is available. You can then use the 
+> `help()` function to see an explanation of the function you're interested in, 
+> for example: `help(numpy.cumprod)`.
+
 {: .callout}
```

Comment:
@drakeasberry thanks for updating the text :).  A couple small things:

1. I noticed that the Travis build is failing on the PR. This is happening because when you push changes, it automatically triggers a `make lesson-check` command to be run (to verify everything still works as expected) and currently this is giving an error. That should go away if you remove the blank line on line 400 which is currently causing the issue.
2. If you could add `<kbd>` tags around "Tab" on line 397, (`<kbd>Tab</kbd>`) this will add the preferred formatting for keyboard keys when rendered (I couldn't recall how to do that formatting, so I'm glad Maxim included it!).